src/System.Management.Automation/engine/regex.cs

```diff
@@ -49,24 +49,22 @@ public enum WildcardOptions
     /// </summary>
```

Comment:
@daxian-dbw, your last commit had 38 failures in `PowerShell-CI-linux`
(These are 5 of the failures)

SemanticVersion api tests.Error handling.Missing parts: 'null': '$null'
```powershell
Method invocation failed because [System.Management.Automation.LanguagePrimitives] does not contain a method named 'IsNull'.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/engine/Basic/SemanticVersion.Tests.ps1: line 207
207:             if ([LanguagePrimitives]::IsNull($version)) {

```
SemanticVersion api tests.Error handling.Missing parts: 'NullString': ''
```powershell
Method invocation failed because [System.Management.Automation.LanguagePrimitives] does not contain a method named 'IsNull'.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/engine/Basic/SemanticVersion.Tests.ps1: line 207
207:             if ([LanguagePrimitives]::IsNull($version)) {

```
SemanticVersion api tests.Error handling.Missing parts: 'EmptyString': '<empty>'
```powershell
Method invocation failed because [System.Management.Automation.LanguagePrimitives] does not contain a method named 'IsNull'.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/engine/Basic/SemanticVersion.Tests.ps1: line 207
207:             if ([LanguagePrimitives]::IsNull($version)) {

```
SemanticVersion api tests.Error handling.Missing parts: '-'
```powershell
Method invocation failed because [System.Management.Automation.LanguagePrimitives] does not contain a method named 'IsNull'.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/engine/Basic/SemanticVersion.Tests.ps1: line 207
207:             if ([LanguagePrimitives]::IsNull($version)) {

```
SemanticVersion api tests.Error handling.Missing parts: '.'
```powershell
Method invocation failed because [System.Management.Automation.LanguagePrimitives] does not contain a method named 'IsNull'.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/engine/Basic/SemanticVersion.Tests.ps1: line 207
207:             if ([LanguagePrimitives]::IsNull($version)) {

```